crates/burn-dataset/src/dataset/mod.rs

```diff
@@ -6,6 +6,7 @@ mod iterator;
 #[cfg(any(feature = "sqlite", feature = "sqlite-bundled"))]
 mod sqlite;
 mod window;
+mod dataframe;
```

Comment:
We will need to put behind a feature flag.

Reply:
yah good point